src/index.js

```diff
@@ -46,6 +46,20 @@ const unexpectedEnzyme = {
       }
     );
 
+    childExpect.exportAssertion(
+      '<ReactWrapper> to have state satisfying <object>',
```

Comment:
Will state always be an object? otherwise it is probably better to allow <any>?

Reply:
React only allows state to be an `object` or `null`. I have changed to allow `<object|null`>